src/monodroid/jni/android-system.h

```diff
@@ -37,10 +37,10 @@ namespace xamarin { namespace android { namespace internal
 		static std::mutex readdir_mutex;
 		static char *libmonoandroid_directory_path;
 #endif
-
-// _WIN32 is defined with _WIN64 so _WIN64 must be checked first.
-#if __SIZEOF_POINTER__ == 8 || defined (_WIN64)
-#define __BITNESS__ "64bit"
+
```

Comment:
this section looks like white space change only?

Reply:
Yeah they were mixed line endings, see: https://github.com/xamarin/xamarin-android/pull/3337#discussion_r301656998

git keeps making this change for me on Windows if I edit this file.

Reply:
ok lets ignore this then :)